src/lib/components/VideoCard/VideoCard.svelte

```diff
@@ -37,43 +37,55 @@
 
   $: title = customTitle;
   $: description = customDescription;
+  $: loadSubscribeButton = false;
 
   afterUpdate(async () => {
     if (youtubeVideoId && (!customTitle || !customDescription)) {
       const snippet = await getYoutubeVideoData(youtubeVideoId);
-      title = snippet.title;
-      description = snippet.description;
+      if (!customTitle) {
+        title = snippet.title;
+      }
+      if (!customDescription) {
```

Comment:
Previously both the `customTitle` and `customDescription` were getting overwritten if one was blank. This way we only overwrite the blank ones.